cli/cmd/profile.go

```diff
@@ -109,6 +136,8 @@ Example:
 				return profiles.RenderProfileTemplate(options.namespace, options.name, controlPlaneNamespace, os.Stdout)
 			} else if options.openAPI != "" {
 				return renderOpenAPI(options, os.Stdout)
+			} else if options.tap != "" {
+				return renderTapOutputProfile(options, controlPlaneNamespace, os.Stdout)
 			}
 
 			// we should never get here
```

Comment:
Yea, I see your point. I was working off of the assumption that there would be no way to reach `return errors.New("Unexpected error")`. If we tried to do that we would need to not provide any options for the profile command, however, `options.validate()` also runs similar checks so we would return an error before we run into the subsequent checks in question.

On the other hand, I see that doing return `errors.New("Unexpected error")` is beneficial because it represents an undesired state for the command. 

That's a long way of saying I don't have a preference 😂I could go either way now that I think about it.